maven-scm-providers/maven-scm-providers-svn/maven-scm-provider-svnexe/src/main/java/org/apache/maven/scm/provider/svn/svnexe/command/checkin/SvnCheckInCommand.java

```diff
@@ -104,7 +110,13 @@ protected CheckInScmResult executeCheckInCommand(
 
     public static Commandline createCommandLine(
```

Comment:
Do we consider this public API, i.e. is overloading necessary here or can I just change the signature of the existing method?